mix/CodeModel.cpp

```diff
@@ -358,7 +360,7 @@ void CodeModel::gasEstimation(solidity::CompilerStack const& _cs)
 {
 	if (m_gasCostsMaps)
 		m_gasCostsMaps->deleteLater();
-	m_gasCostsMaps = new GasMapWrapper(this);
+	m_gasCostsMaps = new GasMapWrapper();
```

Comment:
1. usually we have to set the parent. once the parent is deleted all the children are deleted (this is how works QOBject). This is not possible here because the parent is in an another thread (and it gives warnings).
now we call deleteLater() in the destructor of codemodel.cpp.